__tests__/app.test.js

```diff
@@ -186,8 +186,56 @@ describe("/api/reviews/:review_id/comments", () => {
       .expect(200)
       .then(({ body }) => {
         const { comments } = body;
-        console.log(comments);
         expect(comments).toEqual([]);
       });
   });
 });
+
+describe("Post /api/reviews/:review_id/comments", () => {
+  test("POST 201: should post 1 new object with 2 properties", () => {
+    return request(app)
+      .post("/api/reviews/1/comments")
+      .send({
+        username: "David",
+        body: "my posted comment",
+      })
+      .expect(201)
+      .then(({ body }) => {
+        const comment = body.createdComment[0];
+        expect(comment).toEqual({
+          comment_id: expect.any(Number),
+          body: "my posted comment",
+          review_id: expect.any(Number),
+          author: "David",
+          votes: expect.any(Number),
+          created_at: expect.any(String),
+        });
+      });
+  });
+  test("POST 406: should not post if comment is empty ", () => {
+    return request(app)
+      .post("/api/reviews/1/comments")
+      .send({
+        username: "David",
+        body: "",
+      })
+      .expect(406)
+      .then(({ body }) => {
+        const { msg } = body;
+        expect(msg).toBe("not acceptable");
+      });
+  });
+  test("POST 406: should not post if comment is more than 400 chars", () => {
+    return request(app)
+      .post("/api/reviews/1/comments")
+      .send({
+        username: "David",
+        body: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s w.",
+      })
+      .expect(406)
+      .then(({ body }) => {
+        const { msg } = body;
+        expect(msg).toBe("not acceptable");
+      });
+  });
```

Comment:
These are interesting tests that I haven't considered. However the tests for the core functionality are missing. They are listed below: 

Status 400, invalid ID, e.g. string of "not-an-id"
Status 404, non existent ID, e.g. 0 or 9999
Status 400, missing required field(s), e.g. no username or body properties
Status 404, username does not exist in the DB